server-helpers.js

```diff
@@ -0,0 +1,183 @@
+require("dotenv").config();
+
+const knex = require("knex")({
+  client: "pg",
+  connection: `postgres://${process.env.DB_USER}:${process.env.DB_PASSWORD}@localhost/exquisite_data`,
+});
+
+class UserHelper {
+  static findAuthorById = async id => {
+    try {
+      return knex('authors')
+        .groupBy('id')
+        .select()
+        .having('id', '=', id)
+    } catch (error) {
+      console.error(error.message)
+    }
+  }
+
+ static makeSecureUserResponse = ({
+    id,
+    name,
+    email,
+    bio,
+    created_at,
+    updated_at,
+  }) => {
+    return {
+      id: id,
+      name: name,
+      email: email,
+      bio: bio,
+      created_at: created_at,
+      updated_at: updated_at,
+    };
+  }
+
+  static checkLoginInfo = (info) => {
+    const infoProvided = Object.keys(info)
+    const result = { hasEnoughInfo: false, message: '', userType: ''}
+    if (!infoProvided.includes("password")) {
+      result.message = 'You need to provide a password'
```

Comment:
Won't this always include a key of password even if the value wasn't submitted? Would something like `if (!infoProvided[password])` make more sense? Or maybe I'm misunderstanding?